rasr/command.py

```diff
@@ -2,7 +2,6 @@
 
 import logging
 import shutil
-import subprocess as sp
```

Comment:
I added this to make the subprocess import clearer, since this gets imported via 15: `from i6_core.util import *`

Reply:
another thing that got lost during stashing...

Reply:
guess gotta be more careful next time